writer.go

```diff
@@ -180,3 +181,150 @@ func (w *FilteredLevelWriter) WriteLevel(level Level, p []byte) (int, error) {
 	}
 	return len(p), nil
 }
+
+var triggerWriterPool = &sync.Pool{
+	New: func() interface{} {
+		return new(bytes.Buffer)
```

Comment:
What about initializing the buffer with a minimum size to avoid initial allocations?